actions/views/drafts.ts

```diff
@@ -74,7 +96,12 @@ export function updateDraft(key: string, value: PostDraft|null, rootId = '', sav
         if (syncedDraftsAreAllowedAndEnabled(state) && save && updatedValue) {
             const connectionId = getConnectionId(state);
             const userId = getCurrentUserId(state);
-            await upsertDraft(updatedValue, userId, rootId, connectionId);
+
+            try {
+                await upsertDraft(updatedValue, userId, rootId, connectionId);
+            } catch (error) {
+                return error;
```

Comment:
is this a valid return value?
I know we are not super-consistent with the return values here, but we normally use

```tsx
return {error};
```

or 
```tsx
return {
    data: false,
    error,
};
```